lib/class-wp-rest-shortcodes-controller.php

```diff
@@ -107,9 +100,25 @@ public function get_shortcode_output( $request ) {
 			setup_postdata( $post );
 		}
 
-		// Since the [embed] shortcode needs to be run earlier than other shortcodes,
-		// do_shortcode() will not work with [embed] if there are other shortcodes registered.
-		if ( has_shortcode( $shortcode, 'embed' ) ) {
+		$output = trim( apply_filters( 'the_content', $shortcode ) );
+
+		if ( empty( $output ) ) {
+			$data['html'] = __( 'Sorry, couldn\'t render a preview', 'gutenberg' );
+			return rest_ensure_response( $data );
+		}
+
+		ob_start();
+		wp_head();
+		$head_scripts_styles = ob_get_clean();
+
+		ob_start();
+		wp_footer();
+		$footer_scripts_styles = ob_get_clean();
+
+		// Check if shortcode is returning a video. The video type will be used by the frontend to maintain 16:9 aspect ratio.
```

Comment:
I don't know why this is still here, but it doesn't seem to offer anything over applying the `the_content` filter. Can you tell me about why you prefer this method?

Reply:
@Lewiscowles1986 Are you referring to why I'm determining the `type`?  I'm finding out the type since sandboxing video content sets the height, as mentioned <a href="https://github.com/Lewiscowles1986/gutenberg/blob/ebee8e26c086abe247b18e6d5de7f3dba62856d0/components/sandbox/index.js#L85">here</a>. I see that you're always returning the `type` value as html - so this <a href="https://github.com/Lewiscowles1986/gutenberg/blob/ebee8e26c086abe247b18e6d5de7f3dba62856d0/components/sandbox/index.js#L87">condition</a> is never met?

Reply:
That condition isn't for `shortcode` component though. You're correct that condition of the `sandbox` component isn't met, because there is no certainty the front-end will enforce a 16:9 aspect ratio. For me it's all about trying to get as close to 1:1 expectations